catalog/USGS/NLCD_RELEASES/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_COVER.jsonnet

```diff
@@ -337,6 +308,11 @@ local self_url = catalog_subdir_url + base_filename;
       maximum: 100.0,
       'gee:estimated_range': false,
     },
+    rangeland_tree: {
+      minimum: 0.0,
```

Comment:
I know it's done elsewhere, but since this an unsigned int8 band (0..255 integers), drop the `.0`. Your choice if you want to fix the other one or not. If you don't, I can clean that up later.

```
      minimum: 0.0,
      maximum: 100.0,
```

e.g. from the code editor:

```
bands: List (9 elements)
0: "rangeland_annual_herbaceous", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHEROI…
1: "rangeland_bare_ground", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHEROID["WGS…
2: "rangeland_non_sagebrush_shrub", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHER…
3: "rangeland_herbaceous", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHEROID["WGS …
4: "rangeland_litter", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHEROID["WGS 84",…
5: "rangeland_sagebrush", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHEROID["WGS 8…
6: "rangeland_shrub", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHEROID["WGS 84", …
7: "rangeland_perennial_herbacous", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84",      DATUM["WGS_1984",        SPHER…
8: "rangeland_tree", unsigned int8, PROJCS["Albers Conical Equal Area",    GEOGCS["WGS 84
```

Reply:
Fixed - removed .0's for all min and max's